### ISSUE TITLE
Accessing User Query Parameter

### ISSUE THREAD

**Issue by ThomasKEllis** [Avaiga/taipy#492]
I've been looking for way to access the query parameters from the user. i.e. access the string from the url after a question mark.
http://127.0.0.1:5000/?status=1

I've been digging around taipy documentation without much luck. Even tried going the flask route so I could use the Flask request.args function. But to no avail.

I think I must be missing something obvious.

`from flask import Flask, request
from taipy import Gui

flask_app = Flask(__name__)

text = "Original text"

# Definition of the page
page = """

<|Run local|button|on_action=on_button_action|>
"""

def on_button_action(state):
    state.text = "Button Pressed"
    first_status = request.args.get("status")
    statuses = request.args.getlist("status")
    print(first_status)
    print(request.args)
    print(statuses)

@flask_app.route("/home")
def home_page():
    first_status = request.args.get("status")
    statuses = request.args.getlist("status")
    print(first_status)
    print(request.args)
    return "The home page."

gui = Gui(page=page, flask=flask_app)
gui.run()`

**Comment by FlorianJacta** [comment]
When you want to read a value passed as a URL parameter (e.g., `[Taipy App URL]?parameter=value`), you can use the *params* parameter of the *[on_navigate](https://docs.taipy.io/en/latest/manuals/gui/callbacks/#navigation-callback)* ([doc](https://docs.taipy.io/en/latest/manuals/reference/taipy.gui.Gui/)) callback provided by Taipy. For example, in this code below, the parameters given in the URL will be printed thanks to this callback:

On Taipy 3.0:
```python
from taipy.gui import Gui
from math import cos, exp

value = 10

page = """
Markdown
# Taipy *Demo*

Value: <|{value}|text|>

<|{value}|slider|>

<|{compute_data(value)}|chart|>
"""

def compute_data(decay:int)->list:
    return [cos(i/6) * exp(-i*decay/600) for i in range(100)]

def on_navigate(state, page_name, params):
    print(params)
    return page_name

Gui(page).run()
```

**Comment by ThomasKEllis** [comment]
Perfect! thanks so much for that. I clearly need to familiarise myself with more of these taipy callback functions.

**Comment by FabienLelaquais** [comment]
This was addressed in PR [taipy-gui#949](https://github.com/Avaiga/taipy-gui/pull/949).
